admin_manual/occ_command.rst

```diff
@@ -1420,8 +1420,8 @@ report showing how many users you have, and when a user was last logged in::
   user:report                         shows how many users have access
   user:resetpassword                  Resets the password of the named user
   user:setting                        Read and modify user settings
-  user:keys:verify                    Verify that the stored public key matches
-                                      the stored private key
+  user:sync-account-data              sync user backend data to accounts table for configured users
```

Comment:
```suggestion
  user:sync-account-data              Sync user backend data to accounts table for configured users
```

Reply:
I used the value from `occ --help` to have it consistent 👀